src/main/haskell/kore/src/Kore/AST/Builders.hs

```diff
@@ -71,7 +71,7 @@ It can also be used to transform a symbol or alias sentence to something that
 can be applied to patterns.
 -}
 applyPS
-    :: SentenceSymbolOrAlias s
+    :: (SentenceSymbolOrAlias s, Show level)
```

Comment:
Why is this "Show level" needed?

Reply:
`deriveShow1` and the like functions require class constraints on all types of kind `*`, even phantom types. With `PolyKinds` on, `level` couldn't be inferred to kind `*` because it's phantom, but not it defaults to `*`.

Reply:
That will go away if we put Object and Meta in their own kind rather than just being dummy types in `*`

Reply:
@bmmoore Right, I have a branch for that like we discussed, but it is waiting on this and on #164.